tests/page/page-request-intercept.spec.ts

```diff
@@ -194,4 +192,43 @@ it('should intercept multipart/form-data request body', async ({ page, server, a
   ]);
   expect(request.method()).toBe('POST');
   expect(request.postData()).toContain(fs.readFileSync(filePath, 'utf8'));
-});
\ No newline at end of file
+});
+
+it('should fulfill intercepted response using alias', async ({ page, server, isElectron, isAndroid }) => {
+  it.fixme(isElectron, 'error: Browser context management is not supported.');
+  it.skip(isAndroid, 'The internal Android localhost (10.0.0.2) != the localhost on the host');
+  await page.route('**/*', async route => {
+    const response = await route.fetch();
+    await route.fulfill({ response });
+  });
+  const response = await page.goto(server.PREFIX + '/empty.html');
+  expect(response.status()).toBe(200);
+  expect(response.headers()['content-type']).toContain('text/html');
+});
+
+it('should intercept with url override', async ({ page, server, isElectron, isAndroid }) => {
+  it.fixme(isElectron, 'error: Browser context management is not supported.');
+  it.skip(isAndroid, 'The internal Android localhost (10.0.0.2) != the localhost on the host');
+  await page.route('**/*.html', async route => {
+    const response = await route.fetch({ url: server.PREFIX + '/one-style.html' });
+    await route.fulfill({ response });
+  });
+  const response = await page.goto(server.PREFIX + '/empty.html');
+  expect(response.status()).toBe(200);
+  expect((await response.body()).toString()).toContain('one-style.css');
+});
+
+it('should intercept with post data override', async ({ page, server, isElectron, isAndroid }) => {
+  it.fixme(isElectron, 'error: Browser context management is not supported.');
+  it.skip(isAndroid, 'The internal Android localhost (10.0.0.2) != the localhost on the host');
+  const requestPromise = server.waitForRequest('/empty.html');
+  await page.route('**/*.html', async route => {
+    const response = await route.fetch({
+      data: { 'foo': 'bar' },
+    });
+    await route.fulfill({ response });
```

Comment:
add a test where it it's fulfilled with overrides too

Reply:
Yeah, I'm not sure why it was not covered, do you want to look at it?